src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_SmartParkAndRideScreen.tsx

```diff
@@ -105,11 +107,52 @@ const Profile_SmartParkAndRideScreenContent = () => {
             <ThemeText>{t(SmartParkAndRideTexts.add.max)}</ThemeText>
           </View>
         )}
+
+        <HowItWorksSection />
       </View>
     </FullScreenView>
   );
 };
 
+const HowItWorksSection = () => {
+  const {t} = useTranslation();
+  const navigation = useNavigation<RootNavigationProps>();
```

Comment:
In general, we've tried to keep all useNavigation hooks on the root level of screens and pass a callback to components.

Reply:
Ah, right. Moving it! 😁